src/Provider/GovUkAccountUser.php

```diff
@@ -28,7 +28,7 @@ public function getField(string $key)
         return $this->data[$key] ?? null;
     }
 
-    public function jsonSerialize(): array
+    public function jsonSerialize(): mixed
```

Comment:
```suggestion
    #[\ReturnTypeWillChange]
    public function jsonSerialize(): array
```

Approved, but would this be a better solution while maintaining a more defined return type?